news/garage.rst

```diff
@@ -109,18 +109,21 @@ obvious reason.**
 NBE garage, F-building
 ----------------------
 
-We are piloting also a data & science garage at NBE every first
-wednesday of the month, room F336 13:00. NBE garage will also cover
+NBE garage used to happen in person every first
+wednesday of the month, room F336 13:00. NBE garage covers also
 issues related to working with personal data, research ethics, best
-practices in biomedical research.
+practices in biomedical research. 
```

Comment:
```suggestion
practices in biomedical research**. 
```
Made this bold, since it quite important I think.  Should we also say "for anyone at Aalto" - do you want others to find this?

(somehow it wouldn't let me do these two changes at once)